### PR TITLE
Port existing BrayCurtis Distance example to meta

### PR DESCRIPTION
**Pull request by abhinavrai44**

### CODE REVIEW

**Comment by karlnapf** [comment]
Cool thanks!
Let's not port *all* the distance examples please :)
There are many other methods that would be better exposed.

Also, your data version is not yet merged (I need to do that first)

**Comment by karlnapf** [comment]
Just merged the data commit, restarting travis

**Comment by karlnapf** [comment]
Ping me if green

**Comment by karlnapf** [comment]
Merge conflicts, needs a rebase. Otherwise good

File: doc/cookbook/source/examples/distance/braycurtis.rst [hunk 1]
```diff
@@ -0,0 +1,39 @@
+======================
```
Comment (karlnapf) [inline doc/cookbook/source/examples/distance/braycurtis.rst]: minor: The "===" are longer than the text, which will cause warnings in the pipeline.

**Comment by karlnapf** [comment]
Thanks @abhinavrai44 
Can you please send integration test data and another PR here to update the data version. Testing for the distances are way more important than having more cookbooks....

@vigsterkr I usually make sure the data is included before merging. BTW do you think we should merge more distance cookbooks? I would rather only have the code/test to avoid the cookbook being spammed with 100 distance examples?

**Comment by vigsterkr** [comment]
@karlnapf i've merge the thing without checking the data part but now it's fixed... so no need to do anything now.
but yeah i fully agree we shouldn't do too much of these distance examples as they are all the same....

**Comment by karlnapf** [comment]
What do you mean by "but now its fixed"?

**Comment by vigsterkr** [comment]
i've updated in one fix the data commit hash in the repo :)

**Comment by karlnapf** [comment]
ah cool :)

**Pull request merged**